src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.cs

```diff
@@ -4,63 +4,63 @@
 
 namespace System.Device.Gpio.Drivers
 {
-    /// <summary>
-    /// A GPIO driver for the Raspberry Pi 3.
-    /// </summary>
-    public partial class RaspberryPi3Driver  // Different base classes declared in RaspberryPi3Driver.Linux.cs and RaspberryPi3Driver.Windows.cs
-    {
-        /// <summary>
-        /// Raspberry Pi 3 has 28 GPIO pins.
-        /// </summary>
-        protected internal override int PinCount => 28;
+	/// <summary>
+	/// A GPIO driver for the Raspberry Pi 3.
+	/// </summary>
+	public partial class RaspberryPi3Driver  // Different base classes declared in RaspberryPi3Driver.Linux.cs and RaspberryPi3Driver.Windows.cs
+	{
+		/// <summary>
+		/// Raspberry Pi 3 has 28 GPIO pins.
+		/// </summary>
+		protected internal override int PinCount => 28;
 
-        private void ValidatePinNumber(int pinNumber)
-        {
-            if (pinNumber < 0 || pinNumber > 27)
-            {
-                throw new ArgumentException("The specified pin number is invalid.", nameof(pinNumber));
-            }
-        }
+		private void ValidatePinNumber(int pinNumber)
+		{
+			if (pinNumber < 0 || pinNumber >= PinCount)

```

Comment:
Please fix your environment so that it uses whitespaces instead of tabs, most of the diffs in your PR are simply changes between tabs and whitespaces.

Reply:
apologies on this, i had assumed the .editorconfig file had handled this, I'll see why this happened and correct